### PR TITLE
Loginmodal

### PR DESCRIPTION
**Pull request by JaqKent**
basic login modal

### CODE REVIEW

File: src/components/CustomButton/index.tsx [hunk 1]
```diff
@@ -1,3 +1,4 @@
+/* eslint-disable react/no-unknown-property */
```
Comment (nathan-hyan) [inline src/components/CustomButton/index.tsx:1]: Why is this necessary?

File: src/components/LogInModal/index.tsx [hunk 2]
```diff
@@ -0,0 +1,68 @@
+/* eslint-disable jsx-a11y/no-static-element-interactions */
```
Comment (nathan-hyan) [inline src/components/LogInModal/index.tsx:1]: Why is this necessary?

File: src/components/LogInModal/index.tsx [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+                  {' '}
+                  <span className={styles.link} onClick={link}>Iniciar sesión</span>
+                </p>
+              )}
+
+          </div>
+        </div>
+
+      </div>
+    ) : <></>
```
Comment (nathan-hyan) [inline src/components/LogInModal/index.tsx:64]: Instead of using a ternary, you can use `signIn && <... />`

File: src/components/LogInModal/styles.module.scss [hunk 4]
```diff
@@ -0,0 +1,66 @@
+@import 'scss/colors';
+@import 'scss/mixins';
+@import 'scss/variable';
+
+.container{
+    @include flex(flex-start, center, column);  
+    background: $white;
+    border-radius: 4px;
+    box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.5), 0 6px 20px 0 rgba(0, 0, 0, 0.5);
```
Comment (nathan-hyan) [inline src/components/LogInModal/styles.module.scss:9]: Add shadow as a variable, pretty sure it'll be useful later

File: src/components/LogInModal/styles.module.scss [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+    .close{
+        @include flex(flex-end, flex-end, row);
+        color:$red;
+        cursor: pointer;
+        font-size: 2rem;
+        font-weight: bold;
+        margin-top: 1.5rem;
+        padding-left: 29rem;
+       
+
```
Comment (nathan-hyan) [inline src/components/LogInModal/styles.module.scss:23]: remove empty space

File: src/components/LogInModal/styles.module.scss [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+            padding: 1rem .3rem;
+            width: 4rem;
+        }
+    }
+  .log{
+    @include flex(flex-start, flex-start, row);
+    font-size: 1.2rem;
+    padding-left: 2rem;
+    width: 250px;
+    .link{
```
Comment (nathan-hyan) [inline src/components/LogInModal/styles.module.scss:57]: Format this pls

File: src/components/NavBar/DropdownMenu/index.tsx [hunk 7]
```diff
@@ -1,38 +1,73 @@
 import { useState } from 'react';
 import CustomButton from 'components/CustomButton';
 import profilePic from 'assets/profilepic.jpg';
+import LogInModal from 'components/LogInModal';
```
Comment (nathan-hyan) [inline src/components/NavBar/DropdownMenu/index.tsx:4]: sort imports

File: src/components/NavBar/DropdownMenu/index.tsx [hunk 8]
```diff
@@ -1,38 +1,73 @@
...
   };
+
+  const openLogin = () => {
+    setShowModal(true);
+    setSign(true);
+  };
   const handleLogin = () => {
     setLogged(true);
   };
+  const handleSignin = () => {
```
Comment (nathan-hyan) [inline src/components/NavBar/DropdownMenu/index.tsx:26]: Sign in is the same as login, change it for `Sign up`

File: src/components/NavBar/DropdownMenu/index.tsx [hunk 9]
```diff
@@ -1,38 +1,73 @@
...
   const handleLogOut = () => {
     setLogged(false);
   };
-  const handleSignin = () => {
-    console.log('signin');
+  const handleClose = () => {
+    setShowModal(false);
+  };
+
+  const openSignin = () => {
```
Comment (nathan-hyan) [inline src/components/NavBar/DropdownMenu/index.tsx:36]: Same here

**Review by nathan-hyan — CHANGES_REQUESTED** [review]
Awesome job, some comments

File: src/components/CustomButton/index.tsx (hunk 1 above)
Comment (JaqKent) [reply src/components/CustomButton/index.tsx:1]: deleted, not necesary anymore

File: src/components/LogInModal/index.tsx (hunk 2 above)
Comment (JaqKent) [reply src/components/LogInModal/index.tsx:1]: Avoid non-native interactive elements. If using native HTML is not possible, add an appropriate role and support for tabbing, mouse, keyboard, and touch inputs to an interactive content element.

File: src/components/LogInModal/index.tsx [hunk 10]
```diff
@@ -0,0 +1,68 @@
...
+function LogInModal({
+  signIn, onClick, show, link, onClose,
+} : Props) {
+  return (
+    show ? (
+      <div className={styles.container}>
+        <p className={styles.close} onClick={onClose}>x</p>
+        <div>
+          {signIn
+            ? <h2>Iniciar sesión</h2> : <h2>Crear Cuenta</h2>}
```
[the remark this answers is not in the document]
Comment (JaqKent) [reply src/components/LogInModal/index.tsx:26]: done

File: src/components/LogInModal/index.tsx (hunk 3 above)
Comment (JaqKent) [reply src/components/LogInModal/index.tsx:64]: image

if i use the && i get that error

File: src/components/LogInModal/styles.module.scss (hunk 4 above)
Comment (JaqKent) [reply src/components/LogInModal/styles.module.scss:9]: done

File: src/components/LogInModal/styles.module.scss (hunk 5 above)
Comment (JaqKent) [reply src/components/LogInModal/styles.module.scss:23]: done

File: src/components/LogInModal/styles.module.scss (hunk 6 above)
Comment (JaqKent) [reply src/components/LogInModal/styles.module.scss:57]: done

File: src/components/NavBar/DropdownMenu/index.tsx (hunk 7 above)
Comment (JaqKent) [reply src/components/NavBar/DropdownMenu/index.tsx:4]: moves by itselft at save

File: src/components/NavBar/DropdownMenu/index.tsx (hunk 8 above)
Comment (JaqKent) [reply src/components/NavBar/DropdownMenu/index.tsx:26]: done

File: src/components/NavBar/DropdownMenu/index.tsx (hunk 9 above)
Comment (JaqKent) [reply src/components/NavBar/DropdownMenu/index.tsx:36]: done
